packages/design-system/src/components/Review/__snapshots__/Review.test.tsx.snap

```diff
@@ -18,12 +18,10 @@ exports[`Review renders review 1`] = `
       review text
     </div>
   </div>
-  <div>
```

Comment:
I would kind of expect the removal of this div to have an effect on how it renders in the flex parent because `a` isn't a block element, but it doesn't seem to change anything 🤷. One less div!